homeassistant/components/airnow/__init__.py

```diff
@@ -80,6 +85,13 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
+async def update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Handle options update."""
+    await hass.data[DOMAIN][entry.entry_id].update_radius(
```

Comment:
We could just reload the integration instead

Reply:
What is the best way to do this... `hass.config_entries.async_reload(entry.entry_id)`?

Reply:
Correct